### PR TITLE
test: team booking

### PR DESCRIPTION
**Pull request by shivamklr**
## What does this PR do?

Test team bookings.

[Loom Video](https://www.loom.com/share/ef19ed0a4ad54388b5db765e2e5cbdb0)

## Type of change

- [x] Chore (refactoring code, technical debt, workflow improvements)

## How should this be tested?

- Run `teams.e2e.ts`

## Mandatory Tasks

- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

**Review by shivamklr — COMMENTED** [review]
Self Review

**Review by emrysal — DISMISSED** [review]
I believe you also need to add the member as a host to the team event type for this to work?

**Comment by shivamklr** [comment]
> I believe you also need to add the member as a host to the team event type for this to work?
We are doing that in the user fixture already. Added a new assertion for success page, for clarity. Updated Loom video.

File: apps/web/playwright/teams.e2e.ts [hunk 1]
```diff
@@ -62,4 +64,43 @@ test.describe("Teams", () => {
...
+
+    // Add teamMate2 to the team
+    await prisma.membership.create({
+      data: {
+        teamId: team.id,
+        userId: teamMate2.id,
+        role: MembershipRole.MEMBER,
+        accepted: true,
+      },
+    });
```
Comment (alannnc) [inline apps/web/playwright/teams.e2e.ts:96]: Whats the teamMate2 role in this test? Should we edit event type to be collective event so the success booking have 2 team attendees and 1 guest?

**Review by alannnc — DISMISSED** [review]
Left a comment about teamMember2 role in the test.

File: apps/web/playwright/teams.e2e.ts (hunk 1 above)
Comment (shivamklr) [reply apps/web/playwright/teams.e2e.ts:96]: I think that is a valid point. We should consider a valid schedulingType.

**Review by keithwillcode — APPROVED** [review]

**Pull request merged**